### PR TITLE
Wasp autobuild

### PR DESCRIPTION
**Pull request by skudasov**
This allows us to automatically build and scale load tests in `k8s` both locally and remotely, no other commands needed except `go test -run MyTest`

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -12,7 +12,6 @@ examples/
 
 integration/
 integration-scripts/
-integration-tests/
```
Comment (skudasov) [inline .dockerignore:15]: This is a must since we have a local `replace` I need it to be included if I build `Dockerfile` from the root dir context.

Comment (tateexon) [reply .dockerignore:15]: Which Dockerfile are you using? You can have a dockerignore setup for a specific Dockerfile so we don't have to mess with core's dockerignore.

**Review by tateexon — CHANGES_REQUESTED** [review]

File: .dockerignore (hunk 1 above)
Comment (skudasov) [reply .dockerignore]: Good catch, added default `.gitignore` for WASP image.

File: integration-tests/testconfig/wasp/wasp_autobuild.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package waspconfig
+
+import (
+	"errors"
+)
+
+type WaspAutoBuildConfig struct {
```
Comment (AnieeG) [inline integration-tests/testconfig/wasp/wasp_autobuild.go:7]: How about putting this into CTF ?

**Review by AnieeG — DISMISSED** [review]
please ensure that you have an auto trigger for the test `TestClusterEntrypoint` run . Otherwise rest looks good

**Review by tateexon — APPROVED** [review]

**Review by jmank88 — DISMISSED** [review]

**Review by jmank88 — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Review by AnieeG — APPROVED** [review]

**Pull request merged**